DataManagementSystem/Client/DataManager.py

```diff
@@ -1616,11 +1637,27 @@ def getReplicas( self, lfns, allStatus = True, getUrl = True, diskOnly = False,
 
     result = {'Successful':catalogReplicas, 'Failed':failed}
     if active:
-      self.__checkActiveReplicas( result )
+      self.__filterActiveReplicas( result )
     if diskOnly or preferDisk:
       self.__filterTapeReplicas( result, diskOnly = diskOnly )
+    if forJobs:
+      self.__filterReplicasForJobs( result )
     return S_OK( result )
 
+  def getReplicasForJobs( self, lfns, allStatus = False, getUrl = True ):
```

Comment:
Yes of course, but Chris' recommendation is to split the functionality off the main function for more clarity... This is a attempt to do so. However I didn't remove the additional parameters from the getReplicas() method.